x/ecocredit/client/testsuite/tx.go

```diff
@@ -1426,6 +1427,84 @@ func (s *IntegrationTestSuite) TestTxBuyDirect() {
 	}
 }
 
+func (s *IntegrationTestSuite) TestUpdateProjectAdmin() {
+	val := s.network.Validators[0]
+	updateTo := s.network.Validators[1].Address.String()
```

Comment:
Might be nice to name these `admin` and `newAdmin` rather than `val` and `updateTo`.

Reply:
renamed in https://github.com/regen-network/regen-ledger/pull/1102/commits/d10c329bd8181e2f2129989c49b80e8d7f1673ee